frontend/src/components/authForm/index.js

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import { Link as RouterLink } from 'react-router-dom';
+import {
+	Heading,
+	Card,
+	CardHeader,
+	CardBody,
+	Button,
+	Center,
+	Text,
+	Link,
+} from '@chakra-ui/react';
+import { Form } from 'react-final-form';
+import { validateRegistration, validateLogin } from './validate';
+import InputControl from '../../components/authForm/InputControl';
+
+const AuthForm = ({ type, handleSubmit }) => {
+	return (
+		<Card variant="outline" px={10}>
+			<CardHeader align="center" pb={0}>
+				<Heading>{type}</Heading>
+			</CardHeader>
+			<CardBody pt={2}>
+				<Form
+					onSubmit={handleSubmit}
+					validate={ type == "Register" ? validateRegistration : validateLogin }
+					render={({ handleSubmit, submitting }) => (
+						<form onSubmit={handleSubmit}>
+							{type == 'Register' && (
```

Comment:
`===` for plain comparison and `==` for checking datatype first before comparing
You can read about it [here](https://www.freecodecamp.org/news/javascript-triple-equals-sign-vs-double-equals-sign-comparison-operators-explained-with-examples/#:~:text=Double%20Equals%20(%20%3D%3D%20)%20checks%20for,does%20not%20perform%20type%20coercion.)

```suggestion
							{type === 'Register' && (
```